core/invoker/src/main/scala/whisk/core/container/HttpUtils.scala

```diff
@@ -85,6 +87,9 @@ protected[container] class HttpUtils(
             val containerResponse = Option(response.getEntity).map { entity =>
                 val statusCode = response.getStatusLine.getStatusCode
                 val contentLength = entity.getContentLength
+                if(statusCode == 503 && isProxied) {
```

Comment:
I found this unit tests are using this utility.
`whisk.core.container.test.ContainerConnectionTests`